### PR TITLE
Initial plugin to modify materials and short Mk2 fuselage normal map

### PR DESCRIPTION
**Pull request by blowfishpro**
More normal maps to come but wanted to get plugin and an initial example in first

### CODE REVIEW

File: Distribution/Restock/GameData/ReStock/Patches/FuelTank/restock-fueltanks-mk2.cfg [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@PART[mk2FuselageShortLiquid|mk2FuselageShortLFO|mk2FuselageShortMono]:FINAL
+{
+    MODULE
```
Comment (ChrisAdderley) [inline Distribution/Restock/GameData/ReStock/Patches/FuelTank/restock-fueltanks-mk2.cfg]: Can you add attribution (to yourself) for the content?

Comment (blowfishpro) [reply Distribution/Restock/GameData/ReStock/Patches/FuelTank/restock-fueltanks-mk2.cfg]: Yeah, I'll do that.  Also forgot to make this patch non-final.

**Comment by ChrisAdderley** [comment]
Looks good generally - just the one note about attribution.

**Comment by blowfishpro** [comment]
Force pushed but the changes from the most recent commit are:
* Added a section at the top of the patch file explaining what's in it
* Made patch `:FOR[Restock]` rather than `:FINAL`
* Added myself to the author field of those parts

**Pull request merged**